_posts/2024-08-02-community-news-august-2024.md

```diff
@@ -0,0 +1,61 @@
+
+* [Fix: update hatch install instructions to use the hatch installers rather than pipx in our tutorial #301](https://github.com/pyOpenSci/python-package-guide/issues/301)
+* [All Hatch PRs: [Tuesday july 2 merge] Fix: update hatch install instructions](https://github.com/pyOpenSci/python-package-guide/pull/308)
+
+And for those of you who are interested in the end result, after a robust and lively community discussion, we’ve updated our Get to know Hatch tutorial with the following changes:
+
+1. Suggest users default to the Hatch installers rather than `pipx`.
+2. Suggest Linux users use pipx as the install there is simpler.
+
+It’s important to us to elevate this conversation, because all of our materials are community–developed and go through extensive technical review not only before they’re published for the first time, but also for the entire lifetime of the content! We value openness and transparency, and because of the open discussions with community members, we’ve been able to make it even easier to get started with Python packaging!
+
+## <i class="fa-solid fa-earth-americas"></i> Coming soon: a Spanish translation of the pyOpenSci Python Package Guide
+
+This past May, during our [PyCon US sprints](https://www.pyopensci.org/blog/pyopensci-pyconus-2024-sprints.html), pyOpenSci community member [Felipe Moreno](https://github.com/flpm) submitted [an issue to our Python Package Guide GitHub repository](https://github.com/pyOpenSci/python-package-guide/issues/287), asking if we had considered translating the guide to Spanish. Felipe is an experienced security engineering manager with a strong background in Python software development, data science and user-centered design, who leads the Security Data Science team at [Bloomberg LP](https://www.bloomberg.com/company/). Felipe has worked with other projects, setting up infrastructure support through [Sphinx](https://www.sphinx-doc.org/en/master/) for this kind of work. We were, of course, immediately thrilled with the idea, and took Felipe up on the offer! From there, Felipe built a PR that created a [translation guide for contributors](https://github.com/pyOpenSci/python-package-guide/pull/304), along with another PR that [provides internationalization support](https://github.com/pyOpenSci/python-package-guide/pull/298).
+
+Following Felipe’s initial efforts, several of our SciPy sprint attendees continued to work on translation of the pyOpenSci Python Packaging Guide, and you can see their progress in the [guide repository on GitHub](https://github.com/pyOpenSci/python-package-guide/issues?q=is%3Aopen+is%3Aissue+label%3Ascipy-24). This has resulted in 13 PRs to our packaging guide. Our next steps are to create an editorial team composed of members who are fluent in Spanish/native speakers. This team will be  responsible for reviewing and merging these contributions. Stay tuned for updates on this incredible community-driven project!
```

Comment:
@kierisi i've always thought about doing breakout setnences. i've read this in blog posts where this is a paragraph - and then the writer pulls a sentence or two out and restates it in larger text. do you know what i am talking about? We could do this here.  something like 

> Filipe's efforts have yielded 13 translation pull requests  to our packaging guide

if you don't know what i'm talking about because i'm explaining it poorly i can try to find a screen shot. but i see it in blog posts a lot.

Reply:
I know exactly what you mean! I've heard them referred to as pull quotes, and can start incorporating them.